homeassistant/components/sector_alarm/alarm_control_panel.py

```diff
@@ -0,0 +1,82 @@
+"""The Sector Alarm Integration."""
+import logging
+
+import homeassistant.components.alarm_control_panel as alarm
+import homeassistant.components.sector_alarm as sector_alarm
+from homeassistant.components.sector_alarm import DOMAIN as SECTOR_DOMAIN
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_HOME,
+    STATE_ALARM_DISARMED,
+    STATE_ALARM_PENDING,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Initialition of the platform."""
+    sector_connection = hass.data.get(SECTOR_DOMAIN)
+
+    code = discovery_info[sector_alarm.CONF_CODE]
+    panel_id = discovery_info[sector_alarm.CONF_ALARM_ID]
+
+    async_add_entities([SectorAlarmPanel(sector_connection, panel_id, code)])
+
+
+class SectorAlarmPanel(alarm.AlarmControlPanel):
+    """Get the alarm status, and arm/disarm alarm."""
+
+    def __init__(self, sectorConnect, alarmId, code):
+        """Initialize the service."""
+        self._alarmid = alarmId
```

Comment:
Please use snake case for all your variables, see https://www.python.org/dev/peps/pep-0008/ for more information.